src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -553,6 +554,13 @@ internal FunctionLoadRequest GetFunctionLoadRequest(FunctionMetadata metadata, M
 
                 request.Metadata.Bindings.Add(binding.Name, bindingInfo);
             }
+
+            foreach (var property in metadata.Properties)
+            {
+                // worker properties are expected to be string values
+                request.Metadata.Properties.Add(property.Key, property.Value.ToString());
```

Comment:
Can the value be `null` here?

Reply:
it is possible for it to be null, will add a null check